src/Stratis.Features.FederatedPeg.IntegrationTests/Utils/SidechainFederationNodeRunner.cs

```diff
@@ -48,8 +48,8 @@ public override void BuildNode()
                     options.UseReflectionExecutor();
                 })
                 .UseSmartContractWallet()
-                .AddFederationGateway(new FederatedPegOptions(this.counterChainNetwork))
                 .UseFederatedPegPoAMining()
+                .AddFederationGateway(new FederatedPegOptions(this.counterChainNetwork))
```

Comment:
Sorry about this one, could have been in a separate PR. 

Realised that the ordering was off compared to CirrusPegD and fixed it here.